packages/core/src/Stores/client-store.js

```diff
@@ -1275,7 +1275,7 @@ export default class ClientStore extends BaseStore {
         if (is_maltainvest_account) {
             currency = form_values.currency;
         }
-        const { document_number, document_type, ...required_form_values } = form_values;
+        const { document_number, document_type, document_additional, ...required_form_values } = form_values;
```

Comment:
Why do we need to de-structure this here?

Reply:
If we didn't de-structure that from form_values, it will not send the particular docs to BE and will break FE when creating real account 🙏